.github/ISSUE_TEMPLATE/feature---enhancement-proposal.md

```diff
@@ -9,7 +9,7 @@ assignees: ''
 
 <!--
 Please fill in *all* the questions below and don't remove any of them.
-Proposals not following the template below will be closed immediately.
+Proposals not following the template below will not be considered.
```

Comment:
Apologies for the extraneous change, but this reflects that we do not currently have an automated system which closes issues not following the proposal template.

Reply:
I think that makes it not look "severe" enough. People might not take the warning as seriously :slightly_smiling_face:

Reply:
@Calinou Unless we want to start closing all proposals that don't fill out the template, the old text is incorrect. I think this is a good change.

Reply:
We could keep the old error message and implement a bot that closes issues not following the template.

Reply:
Yes, this reflects the current way of governing proposals. People don't take it seriously in either case, so this makes it more welcoming. Those rules don't mean anything if they are not enforced, don't you agree?

Otherwise we'd also have to close proposals like #2230 by reduz, talking about giving a good example...

See also my proposed change in #2238.